__tests__/utils.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright © 2024 United States Government as represented by the
```

Comment:
```suggestion
 * Copyright © 2023 United States Government as represented by the
```

Just paste the copyright header verbatim from our other files. I never bother updating the copyright date because (a) this is the _exact_ text that the Goddard lawyers told us to use and (b) the copyright may be meaningless in the United States anyway; works of the United States Government are not eligible for domestic copyright protection.